textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -1,6 +1,10 @@
 package edu.uci.ics.textdb.dataflow.queryrewriter;
 
+import edu.uci.ics.textdb.api.common.IField;
```

Comment:
Per our earlier discussion with @rajesh9625 , please use containsAllResults() in your test case.  See DictionaryMatcherTest as an example.